uo_sys_framework/AOLimitModule/cfgFunctions/common/fn_Init.sqf

```diff
@@ -77,15 +77,15 @@ EXEC_CHECK(CLIENT);
             _argNested params ["_unit","_args","_startedInside","_outSide","_enteredZone","_arrayname","_softAOMode","_recheckDead","_pos"];
             _args params ["","_area","","_entryMode","_airsetting","","_softAOtime","_softAOtimeAir"];
 
-            LOG_1("_argNested: %1",_argNested);
-            LOG_1("_area: %1",_area);
-            LOG_1("(vehicle _unit): %1",(vehicle _unit));
```

Comment:
You can just TRACE these

Reply:
They spam the server as they are in an PFH.

Reply:
I would like to run debug on the server